src/applications/disability-benefits/view-payments/containers/helpers.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+export const breadcrumbLinks = [
+  <a href="/" aria-label="Back to VA Home page" key="1">
+    Home
+  </a>,
+  <a
+    href="/view-change-dependents/"
+    aria-label="View your VA payment history"
+    key="3"
+  >
+    View your VA payment history
+  </a>,
+  <a href="/view-change-dependents/view" key="4">
```

Comment:
Should this href hold a different value?

Reply:
Same as above